app/views/restaurants/index.html.erb

```diff
@@ -1,36 +1,28 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Welcome to Eat-Well!</h1>
+<%= react_component("Header", props: { text: "hey", collection: @restaurants }, prerender: false) %>
```

Comment:
There should be a single entry point to the application (not admin), which is `root#main` and `application.html.erb`, which should invoke `<App />` that includes the entire application. Thereafter the application should respond to routes using `react-router` and history. The last part might not be needed in this application but you shouldn't render two components here, just `<App>`, i.e
```
class App extends React.Component {
  render() {
    const {
      restaurants
    } = this.props;

    return (
      <div>
        <Header />
        <List items={restaurats} />
      </div>
    );
  }
}
```